src/modals/edit-profile/profile-form/location-input/nodeGeocoder.js

```diff
@@ -0,0 +1,13 @@
+import nodeGeocoder from 'node-geocoder';
```

Comment:
Perhaps the function I coded above could be put into a `get-location.js` and move this stuff there as well?